Sources/include/module.modulemap

```diff
@@ -5,7 +5,7 @@ module HTMLKit {
 	export *
 
 	explicit module Private {
-		header "CSSCodePoints.h"
+		textual header "CSSCodePoints.h"
```

Comment:
Hey there! The textual header is treated analogous to the `#include` directive, i.e. the header is textually included and is the proper way for including a header multiple times. In this case headers with conflicting defines.